website/content/docs/configuration/ui.mdx

```diff
@@ -15,22 +15,36 @@ more with the Vault UI.
 
 -> The UI requires **Vault 0.10 or higher** or Vault Enterprise.
 
+~> **Vault 1.13** deprecated the `ui` parameter. The UI can now be enabled using
+the `ui_config.enabled` configuration option.
+
```

Comment:
```suggestion
```

Style correction: avoid referencing behavior from previous versions. Focus on current behavior in the current documentation.